Help Desk/CHANGEDB.php

```diff
@@ -421,3 +421,9 @@
 $sql[$count][0]="2.1.03";
 $sql[$count][1]="
 ";
+
+//v2.1.04
+$count++;
+$sql[$count][0]="2.1.04";
+$sql[$count][1]="Update helpDeskIssue Set gibbonPersonID = 0000000001 Where gibbonPersonID = 0000000000;end
```

Comment:
Hi Ali, I've made a suggestion to uppercase the SQL keywords. While it's not required for the query to work, it's a common code style, and something that we aim to do for all queries in Gibbon for consistency. Thanks!

```suggestion
$sql[$count][1]="UPDATE helpDeskIssue SET gibbonPersonID = 0000000001 WHERE gibbonPersonID = 0000000000;end
```